.github/workflows/release.sh

```diff
@@ -205,29 +240,63 @@ _releases_update() {
     if [[ "${GITHUB_REF_NAME}" == main ]]; then
         local filename_no_ext
         filename_no_ext=$(filename_no_ext_for "$1")
+
         local tar_gz_path
         tar_gz_path=$(tar_gz_path_for "$1")
 
         local crc32
         crc32=$(crc32 "${tar_gz_path}")
+
         local date
         date=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
-        echo "${filename_no_ext} ${crc32} ${date}" >>_RELEASES
-        sort -o _RELEASES _RELEASES
 
-        local release_name="release/${filename_no_ext}"
+        touch _RELEASES
+        local update_releases_prefix
+        local release_was_found
+        release_was_found=$(grep "${filename_no_ext} " _RELEASES)
+        if [[ -z "${release_was_found}" ]]; then
+            echo "${filename_no_ext} ${crc32} ${date}" >>_RELEASES
+            update_releases_prefix="add"
+        fi
+
         git config user.name "GitHub Actions"
         git config user.email "actions@user.noreply.github.com"
+
+        local release_name="release/${filename_no_ext}"
+        if [[ ${global_IS_NIGHTLY_OTP} == true ]]; then
+            if [[ -n "${release_was_found}" ]]; then
+                # Replace (inline) in previously existing file (this is a special target)
+                sed -i -e "s|${filename_no_ext} \(.*\)|${filename_no_ext} ${crc32} ${date}|g" _RELEASES
+                update_releases_prefix="replace"
+            fi
+
+            # This is not atomic and might fail, but that's the cost of bleeding edge
+            local git_tag
+            git_tag=$(git_tag_for "$1")
+            gh release delete "${git_tag}" --cleanup-tag --yes || true
+            git branch -D "${release_name}" || true
+            git push origin --delete "${release_name}" || true
```

Comment:
This is the major difference, the "moving target" branch. (mind you, this is untested, as of now)